sonar-plugin-api/src/main/java/org/sonar/api/measures/MeasureBuilder.java

```diff
@@ -19,11 +19,15 @@
  */
 package org.sonar.api.measures;
 
+import org.sonar.api.batch.sensor.SensorContext;
+
 /**
  * MeasureBuilder helps to build complex measures.
  *
  * @since 1.10
+ * @deprecated since 5.6. To create a new measure on scanner side use {@link SensorContext#newMeasure()}
  */
+@Deprecated
 public interface MeasureBuilder {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)